.vscode/launch.json

```diff
@@ -0,0 +1,64 @@
+{
+    "configurations": [
+        {
+            "comments": [
+                "Full launch.json configuration details can be found here:",
+                "https://code.visualstudio.com/docs/cpp/launch-json-reference"
+            ],
+            "name": "DEBUG: (gdb-oneapi) nbody_dpcpp_d Launch",
+            "type": "cppdbg",
+            "request": "launch",
+            "preLaunchTask": "Debug C/C++: DPCPP Makefile",
+            "postDebugTask": "",
+            "program": "/home/pablo/src/n-body/cuda-to-sycl-nbody/nbody_dpcpp_d",
+            "args": [],
+            "stopAtEntry": true,
+            "cwd": "${workspaceFolder}",
+            "environment": [
+                {
+                    "name": "ZET_ENABLE_PROGRAM_DEBUGGING",
+                    "value": "1"
+                },
+                {
+                    "name": "IGC_EnableGTLocationDebugging",
+                    "value": "1"
+                }
+            ],
+            "externalConsole": false,
+            "MIMode": "gdb",
+            "miDebuggerPath": "gdb-oneapi",
+            "setupCommands": [
+                {
+                    "description": "Enable pretty-printing for gdb",
+                    "text": "-enable-pretty-printing",
+                    "ignoreFailures": true
+                },
+                {
+                    "description": "Disable target async",
+                    "text": "set target-async off",
+                    "ignoreFailures": true
+                },
+                {
+                    "description": "Do not display function arguments when printing a stack frame",
+                    "text": "set print frame-arguments none",
+                    "ignoreFailures": true
+                }
+            ]
+        },
+        {
+            "comments": [
+                "Full launch.json configuration details can be found here:",
+                "https://code.visualstudio.com/docs/cpp/launch-json-reference"
+            ],
+            "name": "DEBUG: (cuda-gdb) nbody_cuda_d Launch",
+            "type": "cuda-gdb",
+            "request": "launch",
+            "preLaunchTask": "Debug C/C++: CUDA Makefile",
+            "postDebugTask": "",
+            "program": "/home/pablo/src/n-body/cuda-to-sycl-nbody/nbody_cuda_d",
```

Comment:
Should this reference a specific path on a machine? Is there a way to make this generic?